P: [
  0.0,  0.0,  0.0,  0.25,  -0.1,  0.0,  0.0,
  0.0,  0.0,  0.0,  0.0,  0.25,  -0.1,  0.0,
  0.0,  0.0,  0.0,  0.0,  0.0,  0.25,  0.0
]
replan_cycle:       130
nodePointDistances: [25.0, 50.0, 75.0]
target_rate:        50 # hz

target_speed:       50 # km/h